my-account/app/com/commercetools/sunrise/myaccount/authentication/changepassword/viemodels/ChangePasswordPageContentFactory.java

```diff
@@ -0,0 +1,42 @@
+package com.commercetools.sunrise.myaccount.authentication.changepassword.viemodels;
+
+import com.commercetools.sunrise.framework.viewmodels.PageTitleResolver;
+import com.commercetools.sunrise.framework.viewmodels.content.FormPageContentFactory;
+import com.commercetools.sunrise.myaccount.authentication.changepassword.ChangePasswordFormData;
+import play.data.Form;
+
+import javax.inject.Inject;
+
+public class ChangePasswordPageContentFactory extends FormPageContentFactory<ChangePasswordPageContent, Void, ChangePasswordFormData> {
+
+    private final PageTitleResolver pageTitleResolver;
+
+    @Inject
+    protected ChangePasswordPageContentFactory(final PageTitleResolver pageTitleResolver) {
+        this.pageTitleResolver = pageTitleResolver;
+    }
+
+    @Override
+    protected ChangePasswordPageContent newViewModelInstance(final Void input, final Form<? extends ChangePasswordFormData> form) {
+        return new ChangePasswordPageContent();
+    }
+
+    @Override
+    protected void initialize(final ChangePasswordPageContent viewModel, final Void input, final Form<? extends ChangePasswordFormData> form) {
+        super.initialize(viewModel, input, form);
+        fillChangePasswordForm(viewModel, form);
+    }
+
+    @Override
+    protected void fillTitle(final ChangePasswordPageContent viewModel, final Void input, final Form<? extends ChangePasswordFormData> form) {
+        viewModel.setTitle(pageTitleResolver.getOrEmpty("my-account:changePassword.title")); // TODO messages missing in FE templates
```

Comment:
We have a translation for the title in our [my-account.yaml](https://github.com/commercetools/commercetools-sunrise-theme/blob/master/input/i18n/en/my-account.yaml#L86), but it's called `changePasswordTitle`. We should later rename it to `title`, but for now I would suggest to use `my-account:changePassword.changePasswordTitle` here.